train.py

```diff
@@ -250,6 +259,7 @@ def loss_fn(pred, labels):
     # train loop
     model.train()
 
+    # TODO(whc) Make CheckpointManager accept lists of models, optimizers, schedulers but save a single logical ckpt from them
```

Comment:
So you will need a list of tuple, `List[Tuple[nn.Module, torch.optim.Optimizer, Scheduler]]`. That is important because we need to know which model is optimized with which optimizer to do the parameter ID -> FQN conversion.

Reply:
makes sense.
i guess its still possible to accept 3 separate tuples/lists at the CheckpointManager api, and assert same length, then just match them like 0:0:0, 1:1:1, 2:2:2?

or i need to keep them in a tuple together to give to DCP for some reason

Reply:
yes, you can use 3 lists/tuples as long as CheckpointManager pass the correct pairs to the APIs.